extensions/SemanticMediaWiki/DefaultSettings.php

```diff
@@ -254,30 +254,30 @@
 # or be offered to download "index.php". It might be okay to enable this if
 # such problems are not observed in your wiki.
 ##
-$GLOBALS['smwgLinksInValues'] = false;
+$smwDefaults['smwgLinksInValues'] = false;
 ##
 
 ###
 # Settings for recurring events, created with the #set_recurring_event parser
 # function: the default number of instances defined, if no end date is set;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this commented out code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS125)